Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+node('maven') {
+
+   stage('checkout tfrs src onto docker slave')
+   git url: 'https://github.com/dmarley/tfrs.git'
```

Comment:
shouldn't be using `checkout scm`? it looks like it will checkout from a fork as opposed to the officila/bcgov repository